src/i18n.js

```diff
@@ -56,6 +61,7 @@ i18n
               save: 'Save'
             },
             fileMenu: {
+              label: 'Action menu',
```

Comment:
Not sure I would know what this means as a user - maybe something like `File actions menu` or `Open file menu`